docs/components/modeler/feel/what-is-feel.md

```diff
@@ -4,45 +4,33 @@ title: What is FEEL?
 description: "FEEL is a part of DMN specification of the Object Management Group."
 ---
 
-FEEL (Friendly Enough Expression Language) is a part of
-the [DMN specification](http://www.omg.org/spec/DMN/) of the Object Management Group (OMG). It is designed to write expressions for decision tables and literal expressions in a way that is easily understood by business professionals and developers.
+import { MarkerCamundaExtension } from "@site/src/mdx/MarkerCamundaExtension";
 
-## Unary-tests vs. expressions
+FEEL, which stands for "Friendly Enough Expression Language", is designed to write expressions in a way that is easily understood by business professionals and developers. In Camunda, FEEL is used to define expressions in the context of both BPMN and DMN diagrams. FEEL is a part of
+the [DMN specification](http://www.omg.org/spec/DMN/) of the Object Management Group (OMG).
```

Comment:
Let's link to HTTPS version of the website: https://www.omg.org/spec/DMN

Reply:
added in [2b4be69](https://github.com/camunda/camunda-docs/pull/2771/commits/2b4be69bcc451c400d53caf086dca8a03180f041)